spec/features/signup_page_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+describe 'signing up' do
+  context 'a invalid user' do
+    it 'does not create the user' do
+      visit new_user_registration_url
+      fill_in 'user_name', with: 'Foo Bar'
+      fill_in 'user_password', with: 'password'
+      fill_in 'user_password_confirmation', with: 'password'
+      expect do
+        within('.actions') do
+          click_on 'Sign up'
+        end
+      end
+        .to change { User.count }
```

Comment:
Ouch - Rubocop was okay with this? An `end`, a new line, and then a `.`?

I would prefer

```ruby
expect do
  within('.actions'){ click_on 'Sign up' }
end.to change { User.count }.by 0
```